nexus/src/app/background/tasks/webhook_deliverator.rs

```diff
@@ -141,20 +140,17 @@ impl WebhookDeliverator {
         Self { datastore, nexus_id, cfg, client }
     }
 
-    const MAX_CONCURRENT_RXS: NonZeroU32 = {
-        match NonZeroU32::new(8) {
-            Some(nz) => nz,
-            None => unreachable!(),
-        }
-    };
+    const MAX_CONCURRENT_RXS: usize = 8;
 
     async fn actually_activate(
         &mut self,
         opctx: &OpContext,
         status: &mut WebhookDeliveratorStatus,
     ) -> Result<(), Error> {
-        let mut tasks = JoinSet::new();
-        let mut paginator = Paginator::new(Self::MAX_CONCURRENT_RXS);
+        let mut tasks =
```

Comment:
Same here; I dunno if we have a cap on the "total number of webhook_rx"s on the rack at the moment.